src/universal/conf/faucet.conf

```diff
@@ -24,6 +24,22 @@ faucet {
   # Address of Ethereum node used to send the transaction
   rpc-address = "http://127.0.0.1:8546/"
 
+  # certificate of Ethereum node used to send the transaction when use HTTP(S)
+  certificate = null
```

Comment:
maybe we should group `certificate` and `rpc-address` with something like: `faucet.rpc-client`

Reply:
Done